src/components/controls/checkbox.scss

```diff
@@ -23,7 +24,7 @@
 /* Create a custom checkbox */
 .checkmark {
     position: absolute;
-    top: 0;
+    top: 10px;
```

Comment:
MapLayers checkboxes are placed slightly off, is top: 10px needed?

Reply:
The checkboxes are broken in master, they practically only work if the text in only on one row. This was a bad attempt to fix it. I reverted the change, so they are now back to normal, but still broken.